src/main/java/org/fundacionjala/coding/david/ReverseWords.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.david;
+
+/**
+ * * The class scanner word and reversed.
+ */
+public class ReverseWords {
+
+    /**
+     * This method realize reverse of words.
+     * @param words String.
+     * @return One word reverse.
+     */
+    public String reverseToStartInFivePosition(String words) {
+
+        String[] listWord = words.split(" ");
+        StringBuilder listReverseWord = new StringBuilder();
+        listReverseWord.append(listWord[0]);
+        for (int c = 1; c < listWord.length; c++) {
+            listReverseWord.append(" ");
+            if (listWord[c].length() > 4) {
+                listReverseWord.append(new StringBuilder(listWord[c]).reverse());
+            } else {
+                listReverseWord.append(listWord[c]);
+            }
+        }
+        return listReverseWord.toString();
```

Comment:
Instead of using a StringBuilder adding whitespaces
You can use a StringJoiner
Try this.
```java
        String delimiter = " ";
        String[] listWord = words.split(delimiter);
        StringJoiner listReverseWord = new StringJoiner(delimiter);
        for (int c = 1; c < listWord.length; c++) {
            listReverseWord.add(listWord[c].length() > 4 ? new StringBuilder(listWord[c]).reverse() : listWord[c]);
        }
        return listReverseWord.toString();
```